package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh

```diff
@@ -309,10 +311,13 @@ proto_qmi_setup() {
 
 		uqmi -s -d "$device" --set-client-id wds,"$cid_6" --set-ip-family ipv6 > /dev/null 2>&1
 
+		: "${v6apn:=${apn}}"
+		: "${v6profile:=${profile}}"
```

Comment:
What does the colon do here?

Reply:
I tested with with and without colon and without colon it show 'not found'
`root@OpenWrt:~# LANG=C ./ff.sh

I make script like this
var1=512
var2=256
foa=
: "${foa:=${var1}}"
echo "$foa"
foe=
"${foe:=${var2}}"
echo "$foe"

----------------------------------
with colon
512

without colon
./ff.sh: line 23: 256: not found
256
`

Reply:
@hauke it's a no-op, so the default assignment i variable substitution can take place.

Reply:
thanks for the explanation, I didn't know this syntax.